README.md

```diff
@@ -15,14 +17,32 @@ So play DJ on your phone while listening on Mycroft!
 ### Authorization:
 This Skill uses two different methods of authentication. Both need to be filled in correctly for the **Skill** to function correctly.
 
-#### API connection to your Spotify account
-After installing `mycroft-spotify`, in your [Skill
-settings for Spotify](https://home.mycroft.ai/#/skill) in home.mycroft.ai you will see settings for the Spotify Skill. You will see a username and password field and a 'Connect' button. Ignore the username and password field for now, and click the 'Connect' button. You will be prompted to log in to Spotify, and to authorize Mycroft AI to use your Spotify account using OAuth. This allows Mycroft access to your account details such as Playlists.
+#### Personal Access Token
+
+##### Creating access token
+From the [Spotify developer dashboard](https://developer.spotify.com/dashboard/)
+
+1. Click "CREATE AN APP"
+1. Fill out the create application form
+1. Click on the new app and choose EDIT SETTINGS
+1. Under Redirect URIs add `https://localhost:8888`
+
+More info can be found [here](https://developer.spotify.com/documentation/general/guides/app-settings/).
+
+##### Connecting spotify skill
+After installing `mycroft-spotify`, from the mycroft-core folder run the auth.py script in the mycroft-spotify folder
+
+```
+source venv-activate.sh
+python /opt/mycroft/skills/mycroft-spotify.forslund/auth.py
+```
+
+The script will try to guide you through connecting a developer account to the skill and store the credentials locally.
```

Comment:
another trivial suggestion; if we want, we might want to say where the user can confirm that their credentials and auth link are stored (spotipy config dir?) - TBH i couldn't find it based on docs https://pyxdg.readthedocs.io/en/latest/basedirectory.html#xdg.BaseDirectory.save_config_path

Reply:
Good suggestion, I added some info of where to usually expect the resulting files to be stored